Data/ByteString.hs

```diff
@@ -1625,11 +1626,11 @@ zipWith f ps qs
 --
 -- | A specialised version of zipWith for the common case of a
 -- simultaneous map over two bytestrings, to build a 3rd. Rewrite rules
--- are used to automatically covert zipWith into zipWith' when a pack is
+-- are used to automatically covert zipWith into packZipWith when a pack is
 -- performed on the result of zipWith.
```

Comment:
Since this comment will be rendered in haddocks, could you please wrap identifiers (`zipWith`, `Bytestring`, etc.) into single quotes? To make them hyperlinked to their definitions. 

Given that in future we intend to decomission the rewrite rule, we can probably remove the sentence advertising it already.

Reply:
Done in a73879c , improved capitalization of bytestrings

> Given that in future we intend to decomission the rewrite rule, we can probably remove the sentence advertising it already.

I think so too so I went ahead and removed it